### DISCUSSION TITLE
code blocks have issues with the border radius and look messed up

### DISCUSSION THREAD

**Discussion by literon36** [adityatelange/hugo-PaperMod#1543] [Q&A]
Code blocks on my site look kinda messed up since the border radius doesnt apply correctly and they also seem to have another background with a color nowhere specified(???)
The only things i changed in the css are the colors, I haven't touched anything else.
I'm also using the latest commit of the theme.

Below is an example of such an occurance
![image](https://github.com/adityatelange/hugo-PaperMod/assets/125917933/3c5b0b9a-2d63-4eb3-afb8-b946e9ba27a0)

Did I change something else accidentally or is this a bug?

Thanks in advance!

**Comment by bennesp** [comment]
I have partially solved this by using `{linenos=inline}`
![Screenshot 2024-12-17 alle 15 56 04](https://github.com/user-attachments/assets/1ce1e990-884b-4f32-a666-184aae133308)

"Partially" because you can still clearly see the round angles with different color in the background
